integration/query_frontend_test.go

```diff
@@ -92,16 +97,25 @@ func runQueryFrontendTest(t *testing.T, setup queryFrontendSetup) {
 
 	configFile, flags := setup(t, s)
 
+	// setup tls
+	cmd := exec.Command("bash", "certs/genCerts.sh", "certs", "1")
```

Comment:
I would prefer if we move the script to `docs/` as a reference to generate certificates and we commit the certificates used in integration tests directly into `integration/certs` so that we don't have to generate them each time.

Reply:
Discussed offline. I've been told the general consensus is not to commit certs/keys (even if they're self signed).